src/client/automation/utils/stringify-element.ts

```diff
@@ -0,0 +1,24 @@
+import hammerhead from '../deps/hammerhead';
+
+const htmlUtils        = hammerhead.utils.html;
+const nativeMethods    = hammerhead.nativeMethods;
+
+const MAX_TEXT_CONTENT_LENGTH = 10;
+
+export default function stringifyElement (element: HTMLElement | null): string {
+    if (!element)
+        return '';
+
+    const emptyElement = nativeMethods.cloneNode.call(element);
```

Comment:
Need to test it more carefully.
Take some complex pages (for example, https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) and check how different elements are stringified.
Add client tests for this function.